roles/elasticsearch/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
these tasks require `become: true`

Reply:
here I have broken the deprivileged molecule tests and noticed only today when working on the new pr, will fix